src/main/java/step2/domain/Lotto.java

```diff
@@ -0,0 +1,29 @@
+package step2.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Lotto {
+    private final List<Integer> lottoNums;
+    private static final List<Integer> lottoRange = Stream.iterate(1, n -> n + 1).limit(46).collect(Collectors.toList());
```

Comment:
```suggestion
    private static final List<Integer> lottoRange = Stream.iterate(1, n -> n + 1).limit(46).collect(Collectors.toList());
    
    private final List<Integer> lottoNums;
```

클래스 변수 -> 인스턴스 변수 순으로 작성해주는 것이 컨벤션입니다!
또한 클래스 변수와 인스턴스 변수 사이에 공백을 추가하면 가독성이 상승합니다 :)